sunpy/net/jsoc/tests/test_jsoc.py

```diff
@@ -81,8 +81,8 @@ def test_build_table(client):
     table = responses.build_table()
     assert isinstance(table, astropy.table.Table)
 
-    columns = ['T_REC', 'TELESCOP', 'INSTRUME', 'WAVELNTH', 'CAR_ROT']
-    assert columns == table.colnames
+    columns = ['DATE', 'DATE__OBS', 'TELESCOP', 'INSTRUME', 'WAVELNTH']
+    assert columns == table.colnames[:5]
```

Comment:
Will these always be the order of columns?

Reply:
I think they should be in the same order as it is returned by jsoc client, not sure. 
However, it is outdated, I have updated the tests, now the columns are same as before.